ga4gh/frontend.py

```diff
@@ -595,12 +595,14 @@ def oidcCallback():
     return response
 
 
-# The below paths have not yet been implemented
-
-@app.route('/datasets/<no(search):id>')
+@DisplayedRoute(
+    '/datasets/<no(search):id>',
+    pathDisplay='/datasets/<id>')
 def getDataset(id):
-    raise exceptions.NotImplementedException()
+    return handleFlaskGetRequest(
+        id, flask.request, app.backend.runGetDataset)
 
+# The below paths have not yet been implemented
```

Comment:
we can take out this comment, since there are no more methods in this file that have not been implemented